### ISSUE TITLE
Changes in loss functions 

### ISSUE THREAD

**Issue by parva-jain** [RoboticsClubIITJ/ML-DL-implementation#105]
As we have discussed in the gitter channel that the dimensions of the input matrices are wrongly mentioned in the  loss_function.py, therefore need to be changed. Also, the dimensions of the W and Y vectors are not mentioned. Apart from doc change, there is also an additional 2 is divided in the mean squared error which is not there in original loss function(compared to sklearn's method). So allow me to apply the mentioned changes and show you.
